t/basic.t

```diff
@@ -2481,7 +2499,14 @@ subtest "exeptions are not thrown twice" => sub {
 
   $app->processor->clear_exceptions;
 
-  my (undef, $res) = capture_stderr(sub { $jmap_tester->ua->get($uri) });
+  my (undef, $res) = capture_stderr(sub {
+    $jmap_tester->ua->request(
+      $jmap_tester,
+      HTTP::Request->new(GET => $uri),
+      'jmap',
```

Comment:
This passing 'jmap' is a smell, to me:

a) 'jmap' is effectively just a magic string
b) This one in particular isn't even a JMAP request, so knowing that you need to pass 'jmap' here is relying on the guts of the JMAP::Tester logger object.

At the very least, it feels like the logger should have some generic fallback, if one of the magic strings isn't present. Also, 'jmap' as a string is weird in the first place, since the thing is on a JMAP::Tester object. ('upload' and 'download' are also part of JMAP!)

Reply:
(Update, after reading some JMAP::Tester code: it looks like `jmap` is the default for this, so you could probably just eliminate 'jmap' here, but I think the rest of my comment is still relevant.)

Reply:
This goes away with the new http_X methods.